### DISCUSSION TITLE
TypeScript

### DISCUSSION THREAD

**Discussion by RamazanYilmazz** [OmerAtayilmaz/learning-typescript#10] [Q&A] [answered]
What is the most common used Util Types in TS?

**Comment by OmerAtayilmaz** [comment, marked as answer]
Hi, 
I hope this helps :100: 
here is the Util Types that I use almost every week;
Omit -> used for un-wanted attributes in object
Parameters -> used  to get the parameters of an function etc.
Readonly -> as it is name says; it make the object read-only
Pick -> used for take only specific attributes in object
Exclude -> used for prevent used some parameters
ReturnTypes-> used for check what  will be returned by the specificed function

**Reply by RamazanYilmazz** [reply]
thanks its helpful but can you open the omit a little more?
